papaparse.js

```diff
@@ -1028,9 +1030,21 @@
 			return value;
 		}
 
-		function applyHeaderAndDynamicTyping()
+		function parseTransform(field, value) {
+			if (_config.transform) {
+				if (_config.transform.length === 1) {
```

Comment:
Why is required to test the number of arguments? I think the function should always accept the same number of arguments